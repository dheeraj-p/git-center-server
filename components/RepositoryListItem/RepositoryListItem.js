export default function RepositoryListItem({ name }) {
  return (
    <li className="list-group-item d-flex justify-content-between align-items-center">
      {name}
      <button className="btn btn-outline-success btn-sm">Clone</button>
    </li>
  );
}
