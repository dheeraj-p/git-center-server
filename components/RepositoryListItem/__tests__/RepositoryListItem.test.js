import { shallow } from 'enzyme';
import RepositoryListItem from '../RepositoryListItem';

describe('RepositoryListItem', () => {
  it('should render properly', () => {
    const wrapper = shallow(<RepositoryListItem />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
