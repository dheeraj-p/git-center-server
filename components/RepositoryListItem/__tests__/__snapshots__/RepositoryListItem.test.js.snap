// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RepositoryListItem should render properly 1`] = `
"<li className=\\"list-group-item d-flex justify-content-between align-items-center\\">
  <button className=\\"btn btn-outline-success btn-sm\\">
    Clone
  </button>
</li>"
`;
